sdk/search/Build-SearchPackages.ps1

```diff
@@ -0,0 +1,9 @@
+$scriptDir = Split-Path $MyInvocation.MyCommand.Path -Parent
+$packagesDir = "$scriptDir\..\..\binaries\packages"
+
+if (!(Test-Path -Path $packagesDir)) {
+    mkdir $packagesDir
+}
+
+msbuild "$scriptDir\..\..\build.proj" /t:CreateNugetPackage /p:Scope="sdk\search"
+msbuild "$scriptDir\..\..\build.proj" /t:CreateNugetPackage /p:Scope="sdk\search"
```

Comment:
These commands are duplicated. I think we will want to update the second command to build the data-plane libraries.